service/controller/key/key.go

```diff
@@ -384,15 +388,15 @@ func HostVolumesToVolumeMounts(hostVolumes []v1alpha1.KVMConfigSpecKVMNodeHostVo
 	return volumeMounts
 }
 
-func HostVolumesToVolumes(hostVolumes []v1alpha1.KVMConfigSpecKVMNodeHostVolumes) []corev1.Volume {
+func HostVolumesToVolumes(hostVolumes []v1alpha1.KVMConfigSpecKVMNodeHostVolumes, claimName string) []corev1.Volume {
 	var volumes []corev1.Volume
 
 	for _, hostVolume := range hostVolumes {
 		v := corev1.Volume{
 			Name: hostVolume.MountTag,
 			VolumeSource: corev1.VolumeSource{
-				HostPath: &corev1.HostPathVolumeSource{
-					Path: hostVolume.HostPath,
+				PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{
+					ClaimName: claimName,
```

Comment:
Shouldn't there be a unique PVC name for each PV/host volume? If MC worker 1 has two SSDs, they would both have the same PV name, right? Seems like we need a suffix to tell them apart.

Reply:
oh yeah good catch

Reply:
fixed.